.github/CODEOWNERS

```diff
@@ -58,7 +58,7 @@ exporter/datasetexporter/                                        @open-telemetry
 exporter/dorisexporter/                                          @open-telemetry/collector-contrib-approvers @atoulme @joker-star-l
 exporter/elasticsearchexporter/                                  @open-telemetry/collector-contrib-approvers @JaredTan95 @carsonip @lahsivjar
 exporter/fileexporter/                                           @open-telemetry/collector-contrib-approvers @atingchen
-exporter/googlecloudexporter/                                    @open-telemetry/collector-contrib-approvers @aabmass @dashpole @jsuereth @punya @psx95
```

Comment:
I wonder why only this component was updated in this file and not all 3 components? Is there a githubgen that needs a re-run or something?

Reply:
`make update-codeowners`

Reply:
Done.